docs/_scripts/update_preference_docs.py

```diff
@@ -88,57 +118,98 @@
 
-```bash
-napari --reset
-```
+napari provides a Preferences dialog to manage the settings using a graphical user interface (GUI).
+Importantly, this dialog also allows you to customize napari keyboard shortcuts (keybindings).
 
-## The preferences dialog
+On Windows and Linux, the Preferences dialog can be accessed in the **File** menu. On macOS, it can be
+accessed in the **napari** menu.
 
-Starting with version 0.4.6, napari provides a preferences dialog to manage
-some of the provided options.
+The settings are grouped by sections that are accessible in a list on the left side of the dialog.
 
-{%- for section, section_data in sections.items() %}
+{% for section, section_data in sections.items() %}
 
-### {{ section_data["title"] }}
+## {{ section_data["title"] }} Settings
+
+{{ section_data["description"] }}
 
 ![{{ section }}]({{ images_path }}/preferences-{{ section }}.png)
 
-{% endfor%}
+:::{dropdown} More details on the individual {{ section_data["title"] }} settings
+
+{%   for fields in section_data["fields"] %}
+{%     if fields["ui"] %}
+##### {{ fields["title"] }}
+
+*{{ fields["description"] }}*
+
+* <small>Access programmatically with `SETTINGS.{{ section }}.{{ fields["field"] }}`.</small>
+* <small>Environmental variable: `NAPARI_{{ section.upper() }}_{{ fields["field"].upper() }}`</small>
+* <small>Type: `{{ fields["type"] }}`</small>
+
+{% if section == "shortcuts" and fields["default"] is mapping -%}
+**Default Shortcuts**
+
+| Action | Shortcut |
+|--------|----------|
+{%- for key, value in fields["default"].items() %}
+| {{ key }} | {{ value }} |
+{%- endfor %}
+
+{% else -%}
+* <small>Default: `{{ fields["default"] }}`.</small>
+{%- endif %}
+
+{%     endif %}
+{%   endfor %}
+
+:::
+
+{% endfor %}
 
-### Reset to defaults via UI
+## Reset settings to defaults using the Preferences dialog
 
 To reset the preferences click on the `Restore defaults` button and continue
-by clicking on `Restore`.
+by clicking on `Restore Defaults`.
 
 ![{{ reset }}]({{ images_path }}/preferences-reset.png)
 
-## Sections
+## Overriding settings
 
-The settings are grouped by sections and napari core provides the following:
+napari settings can also be overridden by using *environment variables*.
+The variable names follow a pattern: they start with `NAPARI`,
+followed by the Preference section name, yielding, for example, `NAPARI_APPLICATION`, followed by the setting
+name, yielding, for example, `NAPARI_APPLICATION_CONFIRM_CLOSE_WINDOW`.
+(This specific setting controls whether napari will prompt you to confirm closing the application.)
+You can also find the environmentvariable names for each setting in the descriptions for each of the Preference sections above.
 
-{%- for section, section_data in sections.items() %}
+You can override settings for a single napari session by setting environment variables at launch:
 
-### {{ section_data["title"]|upper }}
+```
+NAPARI_APPLICATION_CONFIRM_CLOSE_WINDOW=False napari
+```
```

Comment:
````suggestion

On Mac and Linux:
```
NAPARI_APPLICATION_CONFIRM_CLOSE_WINDOW=False napari
```
On Windows:
```
set NAPARI_APPLICATION_CONFIRM_CLOSE_WINDOW=False && napari
```
````